test_integration/02_stake_and_mint/utils/anchor.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const BN = require('bn.js');
+const AssertAnchorStateRoot = require('./assert_anchor_stateroot');
+const EventDecoder = require('../../../test/test_lib/event_decoder');
+
+/**
+ * This class represents anchor contract on a chain.
+ */
+class Anchor {
+    /**
+     *
+     * @param sourceWeb3 Web3 of source chain for which this anchor tracks
+     * state root.
```

Comment:
Do we want to indent multi-line comments? E.g. two spaces like code?
```suggestion
     *   state root.
```